irods/manager/user_manager.py

```diff
@@ -57,14 +57,26 @@ def get(self, user_name, user_zone=""):
             raise UserDoesNotExist()
         return iRODSUser(self, result)
 
+    def create_remote(self, user_name, user_zone=""):
```

Comment:
The function name suggests this is for creation of remote users (in the local zone), but the `user_zone` parameter defaults to an empty string. I think the the function name clashes with the design.

What happens if the caller passes the local zone as the zone arg?

Reply:
This is out of date, please see new conversation.